usecases/traverser/near_params_vector.go

```diff
@@ -195,6 +195,9 @@ func (v *nearParamsVector) classFindVector(ctx context.Context, className string
 			return nil, "", errors.New("multiple vectors found, specify target vector")
 		}
 	}
+	if len(res.Vector) == 0 {
+		return nil, "", fmt.Errorf("nearObject search-object with id %v has no vector", id)
```

Comment:
it will always be a case that this is related to `nearObject` and not to `nearVector`? maybe we should change the error message to a more general one like: 
```
"object with id %v has no vector"
```
or
```
"vector not found"
```

Reply:
The function is just for nearObject. It starts like this:

```
func (v *nearParamsVector) classFindVector(ctx context.Context, className string,
	id strfmt.UUID, tenant, targetVector string,
) ([]float32, string, error) {
	res, err := v.search.Object(ctx, className, id, search.SelectProperties{}, additional.Properties{}, nil, tenant)
```

and then gets the vector from the object, so I think the error is ok!